tools/biomoldyn/NEQGamma.xml

```diff
@@ -0,0 +1,125 @@
+<tool id="biomd_neqgamma" name="dcTMD friction correction" version="0.@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>for calculating friction and free energy profiles from TMD ensembles</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.5.2</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">scipy</requirement>
+        <requirement type="package" version="1.19.1">numpy</requirement>
+        <requirement type="package" version="1.1.2">pandas</requirement>
+    </requirements>
+    <command><![CDATA[
+        #for t in $xvgs:
+            echo $t &>> ./xvgs.txt &&
+        #end for
+
+        python '$__tool_directory__/NEQGamma.py' 
+            -i ./xvgs.txt
+            -o outp.txt
+            -ofrict ofrict.txt
+            -vel '$vel'
+            -T '$T'
+            -av '$av'
+            -sigma '$sigma'
+            #if $json:
+                -json '$json'
+            #end if
+
+    ]]></command>
+    <inputs>
+        <param type="data_collection" name="xvgs" label="XVG constraint force files" format="xvg" help="XVG files generated by GROMACS"/>
+        <param type="data" name="json" optional="true" label="Ensemble subgroups" format="json" help="Optional: list of sub-ensembles in JSON format (e.g. produced by the 'Extract clusters' tool). If not specified, results will be calculated for the entire ensemble."/>
```

Comment:
@OlegZharkov more json files. A nice default standard view would be really cool I guess.